projects/storefrontlib/cms-structure/guards/cms-page-guard.service.ts

```diff
@@ -71,11 +69,6 @@ export class CmsPageGuardService {
     state: RouterStateSnapshot
   ): Observable<boolean | UrlTree> {
     return this.cmsService.getPageComponentTypes(pageContext).pipe(
-      map((componentTypes) =>
-        pageContext.id === SMART_EDIT_CONTEXT
-          ? [SMART_EDIT_DUMMY_COMPONENT_TYPE, ...componentTypes]
-          : componentTypes
-      ),
```

Comment:
Corresponding comment: https://github.com/SAP/spartacus/pull/17799/files#r1337085626

Reply:
In addition, we have decided to leave `export const SMART_EDIT_DUMMY_COMPONENT_TYPE = 'SmartEditDummyCMSComponent';` in case someone extends `CmsPageGuardService` and copied the above code.

Reply:
Let's add JSDoc to the original const `SMART_EDIT_DUMMY_COMPONENT_TYPE`:
```ts
/**
 * @deprecated - unused since 6.6. Now `SmartEditLauncher` is responsible for triggering the lazy loading of the `SMART_EDIT_FEATURE`
 */
```

Note: feel free to changed 6.6 to 6.5, depending in which version we plan to release this PR.

Reply:
I've added jsdoc